### DISCUSSION TITLE
Real Time Interview Transcription

### DISCUSSION THREAD

**Discussion by oenu** [Stage-Whisper/Stage-Whisper#28] [Ideas]
Either as a part of Gradio's temporary modal sharing, or as a link hosted by a community member, it would be possible to send a link to a webapp fronend built for interviews. 

Features
- Local Audio Recording
- Live monitoring of transcription
- Split audio into chapters / mark important points
- Consent form

**Comment by petersterne** [comment]
Here's what OpenAI has to say about using whisper for real-time transcription: https://github.com/openai/whisper/discussions/2#discussioncomment-3702403

**Comment by petersterne** [comment]
There's another project that's working on using whisper for real-time transcription though: https://github.com/shirayu/whispering

**Comment by petersterne** [comment]
Another interesting implementation of real-time transcription: https://github.com/davabase/transcriber_app/
